securedrop/Dockerfile

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu@sha256:30204139c6ab96ebd75d72f34db390f28c4decd5e563488b4e485bf979397b67
+
+RUN apt-get update # 2017-11-04
```

Comment:
Why pinning and then updating ?

Reply:
The update is necessary as the published images don't have any package data by default.

Reply:
specifically, https://github.com/tianon/docker-brew-ubuntu-core/blob/5fce3945d95630c2fc03c21ef8665d92bd824642/trusty/Dockerfile#L35

Reply:
oh, of course. Not **upgrade** but **update**, sorry for the noise.